src/quote/quote.service.ts

```diff
@@ -35,15 +35,10 @@ export class QuoteService implements OnModuleInit {
       { name: 'coingecko', enabled: true },
       { name: 'codex', enabled: true },
     ],
-    [BlockchainType.Sei]: [{ name: 'codex', enabled: true }],
-    [BlockchainType.Celo]: [{ name: 'codex', enabled: true }],
-    [BlockchainType.Blast]: [{ name: 'codex', enabled: true }],
-    [BlockchainType.Base]: [{ name: 'codex', enabled: true }],
-    [BlockchainType.Mantle]: [{ name: 'codex', enabled: true }],
-    [BlockchainType.Linea]: [{ name: 'codex', enabled: true }],
-    [BlockchainType.Berachain]: [{ name: 'codex', enabled: true }],
-    [BlockchainType.Coti]: [],
-    [BlockchainType.Iota]: [],
+    [BlockchainType.Hedera]: [
```

Comment:
Add price provider config for Hedera, the price of tokens on hedera will collect from coingecko provider